packages/react/src/themes/teams/components/Text/textVariables.ts

```diff
@@ -1,4 +1,15 @@
 import { TextVariables } from '../../../base/components/Text/textVariables'
+import { StrictColorSchemeMapping, StrictColorScheme } from '../../../../themes/types'
+import { TeamsColorNames } from '../../colors'
+import { generateComponentAreas } from '../../../colorUtils'
+
+const textColorComponentAreas = generateComponentAreas('foreground')
```

Comment:
actually, `generateComponentAreas ` encapsulates quite generic logic - essentially, the intent of this method is to create array of string literals _with items type being inferred from these string literals_, so it won't be just generic `string` type in case of `['foreground']`.

Thus, what if we would create generic method for this, something like 
```jsx
// stringLiteralsArray() method ensures 
// that textColorComponentAreas will be of type ('foreground' | ... )[], instead of string[]
const textColorComponentAreas = stringLiteralsArray('foreground',  ...)`
```